Docs changelog 2025 02 21 (#2991)
neondatabase/website: content/docs/guides/reset-from-parent.md

- You can only reset a branch to the latest data from its parent. Point-in-time resets based on timestamp or LSN are possible using [Branch Restore](/docs/guides/branch-restore), a similar feature, with some differences: branch restore leaves a backup branch and is in general is intended more for data recovery than development workflow.
- This reset is a complete overwrite, not a refresh or a merge. Any local changes made to the child branch are lost during this reset.
- Existing connections will be temporarily interrupted during the reset. However, your connection details _do not change_. All connections are re-established as soon as the reset is done.

> - You cannot reset a [root branch](/docs/reference/glossary#root-branch), such as the `main` branch of your Neon project or a schema-only branch. These branches have not parent branch.

Reviewer: ```suggestion
- Root branches (like your project's `main` branch or schema-only branches) cannot be reset because they have no parent branch to reset to.
```

Reviewer: Consider this revision to make the 'why' logic more explicit
